packages/ui/components/dialog/Dialog.tsx

```diff
@@ -93,7 +93,7 @@ export const DialogContent = React.forwardRef<HTMLDivElement, DialogContentProps
           {type === "creation" && (
             <div>
               <DialogHeader title={title} subtitle={props.description} />
-              <div className="flex flex-col space-y-6">{children}</div>
+              <div className="flex max-h-[95vh] flex-col">{children}</div>
```

Comment:
This breaks the design of other modals. I tested the Test Preview in Routing Forms and when scrolling down the margin below the buttons is gone 

<img width="706" alt="Screenshot 2023-05-22 at 08 18 07" src="https://github.com/calcom/cal.com/assets/30310907/d32e4187-4d8c-4802-94f6-6d23b64a8502">

Should be: 
<img width="626" alt="Screenshot 2023-05-22 at 08 19 46" src="https://github.com/calcom/cal.com/assets/30310907/721a26eb-938b-4068-b1ca-4f63cc858849">